src/Workspaces/Core/Portable/Workspaces.csproj

```diff
@@ -1113,6 +1114,11 @@
     <PublicAPI Include="PublicAPI.Shipped.txt" />
     <PublicAPI Include="PublicAPI.Unshipped.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="VSLangProj2">
+      <HintPath>C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.6\VSLangProj2.dll</HintPath>
```

Comment:
We can't do this here. Workspaces is portable. It can't reference VS-specific assemblies.